src/resources/employees.js

```diff
@@ -1,18 +1,26 @@
 import createResourceDescription from './util/createResourceDescription'
 import { HTTP_VERB_POST } from '../constants'
 
-export const employees = createResourceDescription('employees', {
-  includes: ['user', 'enterprise', 'roles'],
-  filters: ['email', 'external_id', 'state', 'roles', 'match'],
-  headers: {
-    [HTTP_VERB_POST]: {
-      suppress_activation_email: (value) => {
-        if (value !== undefined) {
-          return { 'X-suppress-activation-email': true }
-        }
+export const employees = createResourceDescription(
+  'employees',
+  {
+    includes: ['user', 'enterprise', 'roles'],
+    filters: ['email', 'external_id', 'state', 'roles', 'match'],
+    headers: {
+      [HTTP_VERB_POST]: {
+        suppress_activation_email: (value) => {
+          if (value !== undefined) {
+            return { 'X-suppress-activation-email': true }
+          }
 
-        return undefined
+          return undefined
+        },
       },
     },
   },
-})
+  {
+    favoriteContacts: createResourceDescription('favorite_contacts', {
```

Comment:
Small nitpick,, but shouldn't we limit the methods (GET, POST, DELETE) that are available?

Reply:
What isn't available? Isn't this CRUD?

Reply:
Nope, PATCH isn't available